metrics/cli/lib/firebase/service/firebase_service.dart

```diff
@@ -20,4 +20,19 @@ abstract class FirebaseService extends InfoService {
   /// Deploys Firebase rules, indexes, and functions to the project
   /// with the given [projectId] from the given [firebasePath].
   Future<void> deployFirebase(String projectId, String firebasePath);
+
+  /// Upgrades the Firebase account billing plan.
+  void upgradeBillingPlan(String projectId);
```

Comment:
Would it be great to have here the `FutureOr<void>` return type since we may want to automate such steps and they may become asynchronous? What do you think?